### ISSUE TITLE
baseline-checkstyle needs to be updated for Gradle 4.0

### ISSUE THREAD

**Issue by a-k-g** [palantir/gradle-baseline#183]
I'm running into an issue using this with Gradle 4.0. I think we need to update how the `checkstyle.xml` file location is configured in the `baseline-checkstyle` plugin.

I get the following error
```
* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':subproject:checkstyleMain'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:100)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:70)
        ...
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Unable to find: /<root project dir>/<subproject>/config/checkstyle/checkstyle/checkstyle.xml
        at com.puppycrawl.tools.checkstyle.utils.CommonUtils.getUriByFilename(CommonUtils.java:384)
        at com.puppycrawl.tools.checkstyle.ConfigurationLoader.loadConfiguration(ConfigurationLoader.java:179)
        at com.puppycrawl.tools.checkstyle.ant.CheckstyleAntTask.createRootModule(CheckstyleAntTask.java:410)
        ... 59 more
```

It seems to be looking at the wrong location for some reason.

Adding the following snippet to my `build.gradle` makes it work:

```
    checkstyle {
        configDir = new File("${rootProject.rootDir}/.baseline/")
    }
```

**Comment by uschi2000** [comment]
see https://github.com/palantir/gradle-baseline/pull/184

**Comment by ash211** [comment]
Fixed in #184 please reopen if you see issues when trying the next release with that PR (as of now no release yet contains it)

**Comment by ash211** [comment]
@uschi2000 this can be closed.  Fixed in https://github.com/palantir/gradle-baseline/pull/188
